app/views/users/index.html.erb

```diff
@@ -1 +1,9 @@
-<h1>Here is a list of all users<h1>
\ No newline at end of file
+<div class='flex flex-col gap-8 mt-8'>
+<% @users.each do |user| %>
+    <div class='flex gap-8 m-1'>
+      <img src="<%= user.photo_url%>" alt="<%= user.name%>" class='w-32 h-32 rounded-full' />
+      <h2 class=' text-3xl'> <%= link_to user.name, user_path(user)  %> </h2>
```

Comment:
- [ ] Kindly as requested by the project requirements, the user index view page should match the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png) However your design does not match the wireframe👇🏻 Kindly make sure that the index view page matches the given the user index view page should match the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png).

Your Implementation | Original Design
:---:|:---:
![image](https://github.com/biruk-bereka/Blogify/assets/79058364/f10ed198-8257-4359-882a-391374c6ccdf)| ![image](https://user-images.githubusercontent.com/79058364/233747438-62834c84-a19a-490b-9201-2997b7a318cd.png)


- Do the same for your User show page.

Your Implementation | Original Design
:---:|:---:
![image](https://github.com/biruk-bereka/Blogify/assets/79058364/95428fb8-9a9c-4f54-98cb-d0b59a0a7e96) | ![image](https://github.com/biruk-bereka/Blogify/assets/79058364/88c8ebc3-2cee-4863-b4c8-563ccde3a25e)